### PR TITLE
feat/M2-2916-latest-report-file-name

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx [hunk 1]
```diff
@@ -75,21 +73,18 @@ export const Report = ({ activity, identifiers = [], versions = [] }: ReportProp
 
     setLatestReportError(null);
     try {
-      const { data } = await getLatestReport({
+      const res = await getLatestReport({
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx:76]: pls change variable name for more understandable
```suggestion
      const latestReport = await getLatestReport({
```

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx [hunk 2]
```diff
@@ -75,21 +73,18 @@ export const Report = ({ activity, identifiers = [], versions = [] }: ReportProp
...
+        const base64Str = Buffer.from(res.data).toString('base64');
         const linkSource = getLatestReportUrl(base64Str);
-        const curDate = format(new Date(), DateFormats.YearMonthDayHoursMinutesSeconds);
 
-        download(
-          linkSource,
-          `REPORT_${appletData?.displayName}_${activity.name}_${respondentId}_${curDate}.pdf`,
-          LATEST_REPORT_TYPE,
-        );
+        download(linkSource, fileName || 'Report.pdf', LATEST_REPORT_TYPE);
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx:87]: pls move 'Report.pdf' to const

Comment (kdanilenka32) [reply src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx]: done

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx]: done

**Review by nmialik — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**